lib/utils/downloadTemplateFromRepo.test.js

```diff
@@ -146,9 +146,10 @@ describe('downloadTemplateFromRepo', () => {
     });
 
     it('should parse a valid GitHub URL', () => {
-      const output = parseRepoURL('https://github.com/serverless/serverless');
+      const output = parseRepoURL('https://username@github.com/serverless/serverless');
 
```

Comment:
You added a test case to the existing one, but could you separate private repo case from this one.
We should explicitly support both of private and public.